app/ApiClients/GraylogApi.php

```diff
@@ -120,7 +120,7 @@ public function buildSimpleQuery(?string $search = null, ?Device $device = null)
         return implode(' && ', $query);
     }
 
-    public function getAddresses(Device $device): array
+    public function getAddresses(Device $device): Illuminate\Support\Collection
```

Comment:
This is missing the leading \ like `\Illuminate\Support\Collection`  You can only omit the leading slash when the file is in the global namespace.